samples/msal-browser-samples/VanillaJSTestApp2.0/app/default/auth.js

```diff
@@ -27,7 +27,7 @@ function handleResponse(resp) {
     } else {
         // need to call getAccount here?
         const currentAccounts = myMSALObj.getAllAccounts();
-        if (currentAccounts === null) {
+        if (currentAccounts === null || currentAccounts.length < 1) {
```

Comment:
I think we should make this `currentAccounts && currentAccounts.length < 1`

Reply:
done.

Reply:
made the check `!currentAccounts || currentAccounts.length < 1`